tests/test_contrib/test_pytest_plugin/test_signals.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from django.apps import apps
+from django.core.management import call_command
+from django.db import DEFAULT_DB_ALIAS
+from django.db.models.signals import post_migrate, pre_migrate
+from typing_extensions import Final
+
+# value for ``dispatch_uid`` is needed to disconnect signal receiver
+# registered for testing purposes to which we do not have any reference
+# outside of test function
+DISPATCH_UID: Final = 'test_migrate_signals'
+
+
+@pytest.fixture()
+def _disconnect_receivers():
+    """Disconnect testing receiver of ``pre_migrate`` or ``post_migrate``."""
+    yield
+    main_app_config = apps.get_app_config('main_app')
+    pre_migrate.disconnect(sender=main_app_config, dispatch_uid=DISPATCH_UID)
+    post_migrate.disconnect(sender=main_app_config, dispatch_uid=DISPATCH_UID)
+
+
+def test_post_migrate_muted(migrator):
+    """Ensure ``post_migrate`` signal has been muted."""
+    assert not post_migrate.receivers
```

Comment:
What about `pre_migrate` here?

Reply:
It's not muted, please look at my previous comment

Reply:
Oh, I have missed it, sorry.

Reply:
I decided to mute `pre_migrate` signal receivers, because they were also called e.g. when  `Migrator.reset()` was called